tests/api-watch-diff/test.ml

```diff
@@ -126,19 +161,18 @@ let%expect_test "Removing a type" =
     Api_watch_diff.diff_interface ~reference:ref_signature
       ~current:remove_type_signature
   in
-  Format.printf "%b" result;
-  [%expect {|true|}]
+  Format.printf "%a" pp_diff_list result;
+  [%expect {|[Any]|}]
 
 (* Signature for modify_type.mli:
-   > type t = float
-   > type unused_type = string
+   > type t = int
+   > type unused_type = bool
    > val f : t -> string *)
-
 let modify_type_signature =
   Test_helpers.compile_interface
     {|
-    type t = float
-    type unused_type = string
+    type t = int
+    type unused_type = bool
```

Comment:
I think we might as well make this test even simpler by comparing:
```ocaml
type t = int
```
with
```ocaml
type t = string
```
or something similar.

That being said, I think we should add a separate test that exhibits the behaviour when we change a type that way and it impacts a value!

Reply:
Sure, will fix the tests so that we have both: one with just the type change and also one with the value change due to the type change.